gems/smithy/lib/smithy/anvil/client/templates/endpoint_parameters.erb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# This is generated code!
+
+module <%= namespace %>
+  # Endpoint parameters used to influence endpoints per request.
+  <%= documentation %>
+  EndpointParameters = Struct.new(
+    <%- parameters.each do |param| -%>
+    :<%= param.name %>,
+    <%- end -%>
+    keyword_init: true
+  ) do
+    include Smithy::Client::Structure
+
+    # @api private
+    class << self
+      PARAM_MAP = {
```

Comment:
Is PARAM_MAP still necessary for this to function or is there an alternative?

Reply:
Its not currently used so I've removed it. I think it was legacy from V3.  I don't actually see any usage anymore.